diff -urN md5deep-1.13/Makefile md5deep-1.13.new/Makefile
--- md5deep-1.13/Makefile	2007-07-27 22:46:42.000000000 +0200
+++ md5deep-1.13.new/Makefile	2007-08-26 22:10:02.000000000 +0200
@@ -1,13 +1,13 @@
-
-CC       = gcc
-CFLAGS   = -Wall -W -O3
+OURCC	:= gcc
+CC       = $(OURCC)
+CFLAGS   = -Wall $(OPTFLAGS)
 LINK_OPT = -lm
 
 NAME     = md5deep
 VERSION  = 1.13
 
 # Where we get installed
-PREFIX = /usr/local
+PREFIX = /usr
 
 # You can cross compile this program for Win32 using the MinGW compiler
 # See README for details. Put the $PREFIX used to install MinGW in CR_BASE
@@ -95,7 +95,7 @@
 TIGEROBJ = files-tiger.o hash-tiger.o tiger.o
 tigerdeep: CC += -DTIGER
 tigerdeep: $(OBJ) $(TIGEROBJ)
-	$(CC) $(CFLAGS) $(OBJ) $(TIGEROBJ) -o $@$(SUFFIX) $(LINK_OPT)
+	$(CC) $(LDFLAGS) $(CFLAGS) $(OBJ) $(TIGEROBJ) -o $@$(SUFFIX) $(LINK_OPT)
 
 
 
@@ -107,7 +107,7 @@
 WHIRLPOOLOBJ = files-whirlpool.o hash-whirlpool.o whirlpool.o
 whirlpooldeep: CC += -DWHIRLPOOL
 whirlpooldeep: $(OBJ) $(WHIRLPOOLOBJ)
-	$(CC) $(CFLAGS) $(OBJ) $(WHIRLPOOLOBJ) -o $@$(SUFFIX) $(LINK_OPT)
+	$(CC) $(LDFLAGS) $(CFLAGS) $(OBJ) $(WHIRLPOOLOBJ) -o $@$(SUFFIX) $(LINK_OPT)
 
 
 
@@ -119,7 +119,7 @@
 MD5OBJ = files-md5.o hash-md5.o md5.o
 md5deep: CC += -DMD5
 md5deep: $(OBJ) $(MD5OBJ)
-	$(CC) $(CFLAGS) $(OBJ) $(MD5OBJ) -o $@$(SUFFIX) $(LINK_OPT)
+	$(CC) $(LDFLAGS) $(CFLAGS) $(OBJ) $(MD5OBJ) -o $@$(SUFFIX) $(LINK_OPT)
 
 
 
@@ -131,7 +131,7 @@
 SHA256OBJ = files-sha256.o hash-sha256.o sha256.o
 sha256deep: CC += -DSHA256
 sha256deep: $(OBJ) $(SHA256OBJ)
-	$(CC) $(CFLAGS) $(OBJ) $(SHA256OBJ) -o $@$(SUFFIX) $(LINK_OPT)
+	$(CC) $(LDFLAGS) $(CFLAGS) $(OBJ) $(SHA256OBJ) -o $@$(SUFFIX) $(LINK_OPT)
 
 
 
@@ -142,7 +142,7 @@
 SHA1OBJ = files-sha1.o hash-sha1.o sha1.o
 sha1deep: CC += -DSHA1
 sha1deep: $(OBJ) $(SHA1OBJ)
-	$(CC) $(CFLAGS) $(OBJ) $(SHA1OBJ) -o $@$(SUFFIX) $(LINK_OPT)
+	$(CC) $(LDFLAGS) $(CFLAGS) $(OBJ) $(SHA1OBJ) -o $@$(SUFFIX) $(LINK_OPT)
 
 
 #---------------------------------------------------------------------
@@ -150,7 +150,7 @@
 #---------------------------------------------------------------------
 
 BIN = $(PREFIX)/bin
-MAN = $(PREFIX)/man/man1
+MAN = $(PREFIX)/share/man/man1
 
 install: goals
 	install -d $(BIN) $(MAN)
