--- md5deep-1.9.2/Makefile.orig	2005-11-15 05:37:41.000000000 +0100
+++ md5deep-1.9.2/Makefile	2005-11-18 04:09:50.000000000 +0100
@@ -1,13 +1,13 @@
-
-CC       = gcc
-CFLAGS   = -Wall -O3
+OURCC   := gcc
+CC       = $(OURCC)
+CFLAGS   = -Wall $(OPTFLAGS)
 LINK_OPT = -lm
 
 NAME     = md5deep
 VERSION  = 1.9.3
 
 # Where we get installed
-PREFIX = /usr/local
+PREFIX = /usr
 
 # You can cross compile this program for Win32 using the MinGW compiler
 # See README for details. Put the $PREFIX used to install MinGW in CR_BASE
@@ -145,7 +145,7 @@
 #---------------------------------------------------------------------
 
 BIN = $(PREFIX)/bin
-MAN = $(PREFIX)/man/man1
+MAN = $(PREFIX)/share/man/man1
 
 install: goals
 	install -d $(BIN) $(MAN)
