diff -urN md5deep-1.12/Makefile md5deep-1.12.new/Makefile
--- md5deep-1.12/Makefile	2006-04-03 03:18:07.000000000 +0200
+++ md5deep-1.12.new/Makefile	2006-04-03 19:47:43.000000000 +0200
@@ -1,13 +1,13 @@
-
-CC       = gcc
-CFLAGS   = -Wall -W -O3
+OURCC   := gcc
+CC       = $(OURCC)
+CFLAGS   = -Wall $(OPTFLAGS)
 LINK_OPT = -lm
 
 NAME     = md5deep
 VERSION  = 1.12
 
 # Where we get installed
-PREFIX = /usr/local
+PREFIX = /usr
 
 # You can cross compile this program for Win32 using the MinGW compiler
 # See README for details. Put the $PREFIX used to install MinGW in CR_BASE
@@ -150,7 +150,7 @@
 #---------------------------------------------------------------------
 
 BIN = $(PREFIX)/bin
-MAN = $(PREFIX)/man/man1
+MAN = $(PREFIX)/share/man/man1
 
 install: goals
 	install -d $(BIN) $(MAN)
