diff -urN md5deep-1.10/Makefile md5deep-1.10.new/Makefile
--- md5deep-1.10/Makefile	2006-02-02 04:23:23.000000000 +0100
+++ md5deep-1.10.new/Makefile	2006-02-09 17:54:13.000000000 +0100
@@ -1,13 +1,13 @@
-
-CC       = gcc
-CFLAGS   = -Wall -O3
+OURCC   := gcc
+CC       = $(OURCC)
+CFLAGS   = -Wall $(OPTFLAGS)
 LINK_OPT = -lm
 
 NAME     = md5deep
 VERSION  = 1.10
 
 # Where we get installed
-PREFIX = /usr/local
+PREFIX = /usr
 
 # You can cross compile this program for Win32 using the MinGW compiler
 # See README for details. Put the $PREFIX used to install MinGW in CR_BASE
@@ -149,7 +149,7 @@
 #---------------------------------------------------------------------
 
 BIN = $(PREFIX)/bin
-MAN = $(PREFIX)/man/man1
+MAN = $(PREFIX)/share/man/man1
 
 install: goals
 	install -d $(BIN) $(MAN)
